packages/perseus/src/__tests__/renderer.test.tsx

```diff
@@ -219,19 +219,18 @@ describe("renderer", () => {
             expect(renderer.state.lastUsedWidgetId).toBeNull();
 
             expect(renderer.state.widgetInfo).toStrictEqual(question1.widgets);
-            expect(renderer.state.widgetProps).toMatchInlineSnapshot(`
```

Comment:
I changed this test because it's testing against internal Renderer state which is changing to temporarily store Dropdown's WidgetOptions (vs RenderProps). Once the transforms are gone, it won't need to store this in state at all.